tasks.py

```diff
@@ -9,20 +9,79 @@ def setup_frontend(c):
         c.run("yarn")
 
 
+@task()
+def setup_db(c):
+    print("Running DB setup...")
+
+    with c.cd("backend"):
+        try:
+            c.run("cat .env", hide=True)
+        except Failure as ex:
+            print("ERROR! Please copy backend/.env.sample in backend/.env and set your "
+                  ".env vars")
+            raise ex
```

Comment:
do we need this raise? :) might not be that user friend, we already show a useful message anyway 😊

Reply:
same for the other ones, maybe we could create a utility, something like:

```
run_or_fail(c, "docker -v", error_message="Docker missing...")
```

Reply:
Good idea ;D